subsys/storage/flash_map/flash_map_shell.c

```diff
@@ -0,0 +1,48 @@
+/*
+ * Copyright (c) 2019 Intel Corporation
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+#include <shell/shell.h>
+#include <init.h>
+#include <string.h>
+#include <stdio.h>
+#include <stdlib.h>
+#include <ctype.h>
+#include <flash_map.h>
+#include <logging/log.h>
+
+#define LOG_LEVEL CONFIG_LOG_DEFAULT_LEVEL
+
+LOG_MODULE_REGISTER(flash_map_shell);
+
+extern const struct flash_area *flash_map;
+
+static void fa_cb(const struct flash_area *fa, void *user_data)
+{
+	struct shell *shell = user_data;
+
+	shell_print(shell, "%-4d %-8d %-20s  %-12u %-12zu",
```

Comment:
just nip, not a issue - I think hex values are more useful/readable than decimal. Most of memories has power of 2 internal subdivision. So maybe worth to change this.